src/promptflow-azure/promptflow/azure/operations/_run_operations.py

```diff
@@ -729,8 +731,23 @@ def _get_data_type(_data):
     def _resolve_flow_and_session_id(self, run: Run) -> Tuple[str, Optional[str]]:
         """Resolve flow to remote flow and session id."""
         # for remote flow case, leave session id to None and let service side resolve
+        environment_variables = {}
```

Comment:
why do we need to extract environment varibales for prompty?

Reply:
remove it